README.md

```diff
@@ -927,6 +928,20 @@ Adds a `<group />` that always faces the camera.
 </Billboard>
 ```
 
+#### ScreenSpace
+
+[![](https://img.shields.io/badge/-storybook-%23ff69b4)](https://drei.pmnd.rs/?path=/story/abstractions-screenspace--screen-space-story)
+
+Adds a `<group />` that aligns objects to screen space.
+
+```jsx
+<ScreenSpace
+  depth={1} // Distance from camera
+>
+  <Box>I'm in screen space</Box>
+</ScreenSpace>
+```
```

Comment:
Wouldn't this be view space, rather? What is a use-case for this?

Reply:
This normally referred to screen space, since if the depth=0 you should be exactly at screen plane, the use case, if you want for instance drag an object from canvas in to the scene, you draw it in screen space while it not intersecting with anything...  You can use this for previewing of some buffers also... 

The way we have it vvvv in dx11, is a layer/group/shader modifier, that overrides camera semantics for underlaying shaders, so you can have one camera in the scene another one for let's say ui objects.... 

But i have no clue how to write to semantics in three js, so easy addition...